nio_cli/commands/config.py

```diff
@@ -14,22 +15,23 @@ def config_project(name='.'):
     pk_token = input('Enter PK Token (optional): ')
     ws_host = pk_host.replace('pubkeeper', 'websocket')
 
-    pk_host_replace = "s/^PK_HOST:.*/PK_HOST: {}/".format(pk_host)
-    pk_token_replace = "s/^PK_TOKEN:.*/PK_TOKEN: {}/".format(pk_token)
-    ws_host_replace = "s/^WS_HOST:.*/WS_HOST: {}/".format(ws_host)
-
-    if pk_host:
-        subprocess.call(
-            'sed -i "" "{}" {}'.format(pk_host_replace, env_location),
-            shell=True)
-        subprocess.call(
-            'sed -i "" "{}" {}'.format(ws_host_replace, env_location),
-            shell=True)
-    if pk_token:
-        subprocess.call(
-            'sed -i "" "{}" {}'.format(pk_token_replace, env_location),
-            shell=True)
-
+    nenv = open(env_location, 'r')
+    tmp = tempfile.NamedTemporaryFile(mode='w', delete=False)
+    for line in nenv:
+        if re.search('PK_HOST:', line) and pk_host:
```

Comment:
These regexes should probably start with a `^` to prevent matching from some kind of variable like `STAGE_PK_HOST`